schema/CbsdData.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+  	"description": "Describe the CbsdData object",
+	"type": "object",
+	"required": [
+    	"id",
+    	"registration",
+    	"grants"
+	],
+	"properties":{
+		"id":{
+			"type": "string",
+			"description": "Format cbsd/$CBSD_REFERENCE_ID where $CBSD_REFERENCE_ID is defined as UTF-8($FCC_ID + '/' + sha1($SERIAL_NUMBER)), the SHA-1 hash of the device manufacturer serial number that is unique within the FCC ID namespace scope "
+		},
+		"registration":{
+			"type": "object",
+			"description" : "Contains device installation parameters.",
+			"items" : {
+				"$ref" : "file:CbsdRecordData.schema.json"
```

Comment:
Would the name RegistrationRequest.schema.json more closely match the WINNF-TS-0096 definition of CBSD Data's registrationRequest type than CbsdRecordData.schema.json?

Reply:
yes, because the record can be different from registration request(for example cbsd serial number and user id are not required and mostly the SAS will not provide them, cbsd info should not be in the record) and I think later they can have bigger difference.

Reply:
Sounds good.